elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -459,17 +459,17 @@ elytron.simple-regex-realm-mapper=Definition of a simple RealmMapper that attemp
 elytron.simple-regex-realm-mapper.add=Add operation for the RealmMapper
 elytron.simple-regex-realm-mapper.remove=Remove operation for the RealmMapper
 #Attributes
-elytron.simple-regex-realm-mapper.pattern=The regular expression which must contain at least one capture group to extract the realm from the name.
+elytron.simple-regex-realm-mapper.pattern=The regular expression which must contain at least one capture group to extract the realm from the name. In case when more capture groups are obtained in pattern then first capture group is used.
 elytron.simple-regex-realm-mapper.delegate-realm-mapper=The RealmMapper to delegate to if there is no match using the pattern.
 
 elytron.mapped-regex-realm-mapper=Definition of a RealmMapper implementation that first uses a regular expression to extract the realm name, this is then converted using the configured mapping of realm names.
 # Operations
 elytron.mapped-regex-realm-mapper.add=Add operation for the RealmMapper
 elytron.mapped-regex-realm-mapper.remove=Remove operation for the RealmMapper
 #Attributes
-elytron.mapped-regex-realm-mapper.pattern=The regular expression which must contain at least one capture group to extract the realm from the name.
-elytron.mapped-regex-realm-mapper.delegate-realm-mapper=The RealmMapper to delegate to if there is no match using the pattern.
-elytron.mapped-regex-realm-mapper.realm-map=Mapping of realm name extracted using the regular expression to a defined realm name.
+elytron.mapped-regex-realm-mapper.pattern=The regular expression which must contain at least one capture group to extract the realm from the name. In case when more capture groups are obtained in pattern then first capture group is used.
+elytron.mapped-regex-realm-mapper.delegate-realm-mapper=The RealmMapper to delegate to if the pattern does not match. If no delegate is specified then the default realm on the domain will be used instead. In case when username does not matches pattern and delegate realm-mapper is used then result of delegate-realm-mapper is mapped through realm-map.
+elytron.mapped-regex-realm-mapper.realm-map=Mapping of realm name extracted using the regular expression to a defined realm name. In case when value for mapping is not in the map or in case when realm with name which is result of mapping does not exist in given security domain then default realm is used.
```

Comment:
If the value for the mapping is not in the map or the realm whose name is the result of the mapping does not exist in the given security domain, the default realm is used.